app/assets/javascripts/search.coffee

```diff
@@ -42,3 +42,17 @@ $(document).ready ->
 # Handle presence of Javascript by turning off or on visibility of JS sensitive objects
 $('.js-disabled-object').css 'visibility': 'hidden'
 $('.js-enabled-object').css 'visibility': 'visible'
+
+# Change focus of cursor when search link is clicked on Manage Collections, Manage Variables, or Manage Services pages.
+$ ->
+ $('#search-focus').on 'click', ->
+    document.getElementById('keyword').focus()
```

Comment:
@abbottry Do you think this should use jQuery? It probably works just fine, but the rest of the app uses jQuery for this kind of thing